tests/test_compiler.py

```diff
@@ -17,6 +17,7 @@
 import pandas as pd
 from lux.vis.Vis import Vis
 from lux.vis.VisList import VisList
+import psycopg2
```

Comment:
Can we make the SQL tests into a different file so that the separation between SQL and Pandas tests is more clear? That way, it would make it easier to add a flag that only runs PandasExecutor tests (e.g., when developing locally).

Reply:
On a related note, the line `connection = psycopg2.connect("host=localhost dbname=postgres user=postgres password=lux")` is repeated many times across the tests. We could probably put this inside `conftest.py` as a `@pytest.fixture` (variables that are reused across tests in the same session).